commands/_setBotStatus.py

```diff
@@ -2,18 +2,25 @@
 
 import discord
 
-botStatusList = [
-    discord.Activity(type = discord.ActivityType.listening, name = "Martingales | $help"),
-    discord.Activity(type = discord.ActivityType.watching, name = "Geese Honk | $help"),
-    discord.Activity(type = discord.ActivityType.streaming, name = "EGAD videos | $help"),
-    discord.Activity(type = discord.ActivityType.listening, name = "115 ASMR | $help"),
-    discord.Activity(type = discord.ActivityType.playing, name = "Solidworks | $help"),
-    discord.Activity(type = discord.ActivityType.playing, name = "Among Us | $help"),
-    discord.Activity(type = discord.ActivityType.watching, name = "Crowdmark | $help")
-]
+from commands import _util
+
+
+async def reset_status_list(client: discord.Client):
+    global botStatusList
+    botStatusList = [
+        discord.Activity(type = discord.ActivityType.listening, name = "Martingales | $help"),
+        discord.Activity(type = discord.ActivityType.watching, name = "Geese Honk | $help"),
+        discord.Activity(type = discord.ActivityType.streaming, name = "EGAD videos | $help"),
+        discord.Activity(type = discord.ActivityType.listening, name = "115 ASMR | $help"),
+        discord.Activity(type = discord.ActivityType.playing, name = "Solidworks | $help"),
+        discord.Activity(type = discord.ActivityType.playing, name = "Among Us | $help"),
+        discord.Activity(type = discord.ActivityType.watching, name = "Crowdmark | $help"),
+        discord.Activity(type = discord.ActivityType.watching, name = "{0} Users, {1} Guilds".format(_util.get_member_count(client), _util.get_guild_count(client)))
+    ]
```

Comment:
Oh crap, the prefix won't be defined for each guild lol. Can't fix every issue though ahaha